### PR TITLE
Add infographics back, cleanup, some resposive work

### PR DESCRIPTION
**Pull request by schalkneethling**
@ckprice @pmclanahan @alexgibson r? This is a WIP but, feel free to code review.

### CODE REVIEW

File: src/styles/modules/modal-issue.less [hunk 1]
```diff
@@ -1,10 +1,24 @@
 .modal-issue {
+    margin: 0 auto;
     padding: 10rem;
+    height: auto;
```
Comment (alexgibson) [inline src/styles/modules/modal-issue.less]: This line prevents me from scrolling an issue should its content take up more height than the viewport.

**Comment by alexgibson** [comment]
Not related to this change, but I notice if an issue detail is open, the dots still animate in the background. They could probably be paused at this point, to save on CPU cycles?

File: src/app/lorax/directives/modal-issue.js [hunk 2]
```diff
@@ -77,7 +78,26 @@ define(['angular', 'jquery'], function (angular, $) {
             // in /src/app/lorax/directives/window.js (onBgModeChange) which in turn
             // sets the background color of the issue modal based on the status of
             // the current issue.
-            this._windowService.setBgMode(this._$scope.issue.getStatusDescription(), false);
+            this._windowService.setBgMode(this._$scope.modalIssue.issue.getStatusDescription(), false);
+
+            var figure = $('figure');
```
Comment (alexgibson) [inline src/app/lorax/directives/modal-issue.js]: Very broad selector?

**Comment by alexgibson** [comment]
Looks good to me so far :+1: Please ping me when you want me to take another look.

File: src/styles/modules/modal-issue.less (hunk 1 above)
Comment (schalkneethling) [inline src/styles/modules/modal-issue.less]: Yup, removed that one.

**Comment by schalkneethling** [comment]
"but I notice if an issue detail is open, the dots still animate in the background" - Yes, that is not supposed to happen, there is code that should stop that but, I need to debug and figure out why it is not doing what it is supposed to.

File: src/app/lorax/directives/modal-issue.js (hunk 2 above)
Comment (schalkneethling) [inline src/app/lorax/directives/modal-issue.js]: Generally yes but, I know for a fact that there is only one figure element on the page but, with that said, to be more future proof, it is probably better to just give it an id or class.

**Comment by schalkneethling** [comment]
If this is ok, and adds the infographics back, except ad tracking where there is a known bug, let's merge this and I can move forward. Thanks

**Pull request merged**